src/components/interpolate-colors.js

```diff
@@ -87,25 +87,22 @@ export const interpolatePlasma = interpolateSequentialMulti(schemePlasma);
 export function getCellColors(params) {
   const {
     cellColorEncoding,
-    expressionMatrix, geneSelection,
+    expressionData,
     cellSets, cellSetSelection,
     cellSetColor,
+    attrs,
```

Comment:
```suggestion
    expressionMatrixAttrs,
```

See corresponding suggestion in `HeatmapSubscriber.js`, just to make it more clear what this parameter holds.